lib/Wikia/src/Service/User/Auth/AuthService.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Wikia\Service\User\Auth;
+
+interface AuthService {
```

Comment:
Should this be `AuthorizationService`? `AuthService` to me says Authentication, it's probably better to be explicit